boards/FSM/Src/App/App_AcceleratorPedals.c

```diff
@@ -3,6 +3,20 @@
 #include <stdint.h>
 #include "App_AcceleratorPedals.h"
 
+// TODO: Experimentally determine the encoder values for the primary and
+// secondary APPS when the accelerator is fully pressed #666
+#define PAPPS_ENCODER_FULLY_PRESSED_VALUE 2500U
+#define SAPPS_ENCODER_FULLY_PRESSED_VALUE 1800U
+
+// If encoder exceeds the following values, we reset it back to zero (usually
+// due to underflowing)
+#define PAPPS_ENCODER_RESET_VALUE 3500U
+#define SAPPS_ENCODER_RESET_VALUE 2400U
+
+#define HUNDRED_PERCENT 100.0f
+#define ZERO_PERCENT 0.0f
+#define PERCENT_DEFLECTION 0.03f
```

Comment:
make sure you put `#define` around braces

```suggestion
#define PAPPS_ENCODER_FULLY_PRESSED_VALUE (2500U)
#define SAPPS_ENCODER_FULLY_PRESSED_VALUE (1800U)

// If encoder exceeds the following values, we reset it back to zero (usually
// due to underflowing)
#define PAPPS_ENCODER_RESET_VALUE (3500U)
#define SAPPS_ENCODER_RESET_VALUE (2400U)

#define MAX_PEDAL_PERCENTAGE (100.0f)
#define MIN_PEDAL_PERCENTAGE (0.0f)
#define PEDAL_DEFLECTION_PERCENTAGE (0.03f)
```